apps/experiments/tests/test_models.py

```diff
@@ -1069,6 +1069,31 @@ def test_archived_experiments_are_filtered_out(self):
 
 @pytest.mark.django_db()
 class TestExperimentTrends:
+    def test_get_trend_data_returns_data_from_all_versions(self, experiment):
+        """Test that get_trend_data aggregates traces from all versions of an experiment"""
+        # Create some versions of the experiment
+        version1 = experiment.create_new_version()
+        version2 = experiment.create_new_version()
+
+        curr_time = timezone.now()
+        Trace.objects.create(
+            experiment=experiment, team=experiment.team, status=TraceStatus.SUCCESS, timestamp=curr_time, duration=1
+        )
+        # Trace for version1
+        Trace.objects.create(
+            experiment=version1, team=experiment.team, status=TraceStatus.ERROR, timestamp=curr_time, duration=1
```

Comment:
Does this ever happen? I mean do we attach spans to experiment versions?

Reply:
Yikes, looks like we do actually. That's not great. I think we should probably always use the working version - like we do for sessions.

Probably a separate PR for that though.

Reply:
Hmm yeah maybe we should. We could store the version of the experiment in the metadata then

Reply:
Thanks. Here's a carrot :carrot: